dimelo/load_processed.py

```diff
@@ -14,16 +14,28 @@ def pileup_counts_from_bedmethyl(
     bedmethyl_file: str | Path,
     motif: str,
     regions: str | Path | list[str | Path] | None = None,
+    # window_size: int | None = None,
+    single_strand: bool = False,
 ) -> tuple[int, int]:
     """
-    Extract number of modified bases and total number of bases from the given bedmethyl file
+    Extract number of modified bases and total number of bases from the given bedmethyl file.
```

Comment:
Nitpick: I might say something like, "creates a methylated base pileup" as the intro line. Maybe also briefly explain what a pileup is, if this is intended to be user facing? Or just assume a user knows.